uint8 MANUAL_CTRL = 1
uint8 AUTO_HOVER = 2
uint8 CMD_CTRL = 3
uint8 AUTO_TAKEOFF = 4
uint8 AUTO_LAND = 5
uint8 state
bool landed